express/scripts/gnav.js

```diff
@@ -146,11 +147,55 @@ function loadFEDS() {
 
   const isHomepage = window.location.pathname.endsWith('/express/');
   const isMegaNav = window.location.pathname.startsWith('/express')
-    || window.location.pathname.startsWith('/education');
+  || window.location.pathname.startsWith('/education');
   const fedsExp = isMegaNav
     ? `adobe-express/ax-gnav${isHomepage ? '-homepage' : ''}`
     : 'cc-express/cc-express-gnav';
 
+  const breadCrumbList = [];
+  async function buildBreadCrumbArray() {
+    if (isHomepage) {
+      return;
+    }
+    const capitalize = (word) => word.charAt(0).toUpperCase() + word.slice(1);
+    const buildBreadCrumb = (path, name, parentPath = '') => (
+      { title: capitalize(name), url: `${parentPath}/${path}` }
+    );
+
+    const placeholders = await fetchPlaceholders();
+    const validCategories = ['create', 'feature', 'templates'];
+    const pathSegments = window.location.pathname.split('/')
+      .filter((element) => element !== '')
+      .filter((element) => element !== locale);
+    const localePath = locale === 'us' ? '' : `${locale}/`;
+    let category = pathSegments[1];
+    const secondPathSegment = category.toLowerCase();
+    const pagesShortNameElement = document.querySelector('meta[name="short-title"]');
+    const pagesShortName = pagesShortNameElement ? pagesShortNameElement.getAttribute('content') : null;
+
+    if (getMetadata('hide-breadcrumbs') === 'true'
```

Comment:
Why not checking this condition at the top of the function - getMetadata('hide-breadcrumbs') === 'true'? I think that it'd save the code execution that happens above.